### DISCUSSION TITLE
Incompatibility with newer versions of some libraries

### DISCUSSION THREAD

**Discussion by Jamewsss** [NVIDIA-NeMo/Guardrails#284] [Q&A] [answered]
I am wondering when nemo will be updated to be compatible with newer versions of typer, typing extensions, nestasyncio and so on. Although it creates incompatibility errors on install, nemo seems to work fine with these new library versions.

**Comment by drazvan** [comment, marked as answer]
I'm literally working right now up upgrading and relaxing the dependencies. The `0.7.0` version which will be published later today should fix this.

**Reply by Jamewsss** [reply]
Thank you!
